ImplementationGuide/markdown/KontaktGesundheitseinrichtung/Kontakt_Interaktionen.md

```diff
@@ -82,6 +82,22 @@ Folgende Suchparameter sind für das Bestätigungsverfahren relevant, auch in Ko
 
     Dieser Suchparameter ist für die Umsetzung des IHE QEDm Profils verpflichtend.
 
+1. Der Suchparameter "date-start" MUSS unterstützt werden:
+
+   Beispiele:
+
+    ```GET [base]/Encounter?date-start=lt2020-26-10```
+
+    Anwendungshinweise: Weitere Informationen zur Suche nach "Encounter.period" finden sich in der [FHIR-Basisspezifikation - Abschnitt "Date Search"](http://hl7.org/fhir/R4/search.html#date).
```

Comment:
Sollte hier nicht auf R5 verwiesen werden? da "date-start" Search in R4 nicht gegeben, soweit ich verstanden habe? Ggf. einen expliziten hinweis, dass es hier um ein R5 backport geht?

Reply:
siehe https://github.com/gematik/spec-ISiK-Basismodul/pull/240/files